python/cugraph/cugraph/structure/renumber_wrapper.pyx

```diff
@@ -186,13 +186,18 @@ def renumber(input_df,           # maybe use cpdef ?
                     shuffled_df = input_df
                     edge_counts_32 = make_unique[vector[int]](1, num_local_edges)
 
-                shuffled_major = major_vertices.__cuda_array_interface__['data'][0]
-                shuffled_minor = minor_vertices.__cuda_array_interface__['data'][0]
+                if not transposed:
```

Comment:
checking `transposed` again makes it redundant since it is already done in 179. I believe idea was just to swap the column name which will swap the `src column` and `dst column` if `transpose=True`. However, unless I am misunderstanding something, `major_vertices` and `minor_vertices` should be assigned shuffled_df column after checking for `transposed`. If not , when `transpose=True`  `major_vertices` and `minor_vertices` does not get update. Unless `major_vertices` and `minor_vertices` point to the same address as shuffled_df columns which I doubt.

Reply:
And sg doesn't seem to account for transpose in the current implementation